src/Akeneo/Pim/Automation/DataQualityInsights/back/Infrastructure/Symfony/Resources/config/jobs.yml

```
@@ -144,6 +168,12 @@ services:
             - '@Akeneo\Pim\Automation\DataQualityInsights\Application\Consolidation\PurgeOutdatedData'
             - '@logger'
 
+    akeneo.pim.automation.data_quality_insights.infrastructure.connector.tasklet.schedule_periodic_tasks:
+        class: Akeneo\Pim\Automation\DataQualityInsights\Infrastructure\Connector\Tasklet\SchedulePeriodicTasksTasklet
```

```suggestion
    Akeneo\Pim\Automation\DataQualityInsights\Infrastructure\Connector\Tasklet\SchedulePeriodicTasksTasklet:
```

We now try to not use aliases when possible